frontend/workflows/experimentation/src/start-experiment.jsx

```diff
@@ -0,0 +1,209 @@
+            input: {
+              type: "string",
+              key: "downstreamCluster",
+            },
+          },
+          {
+            name: "Upstream Cluster",
+            value: clusterPair.upstreamCluster,
+            input: {
+              type: "string",
+              key: "upstreamCluster",
+            },
+          },
+        ]}
+      />
+      <Grid container justify="center">
+        <Button text="Next" destructive onClick={onSubmit} />
+      </Grid>
+    </WizardStep>
+  );
+};
+
+const AbortExperimentDetails = () => {
+  const { onSubmit, onBack } = useWizardContext();
+  const abortExperimentData = useDataLayout("abortExperimentData");
+  const abortExperiment = abortExperimentData.displayValue();
+  const update = (key, value) => {
+    abortExperimentData.updateData(key, value);
+  };
+
+  return (
+    <WizardStep error={abortExperimentData.error}>
+      <MetadataTable
+        onUpdate={update}
+        data={[
+          {
+            name: "Percent",
+            value: abortExperiment.percent,
+            input: {
+              type: "number",
+              key: "percent",
+              validation: yup.number().integer().moreThan(-1).lessThan(101),
+            },
+          },
+          {
+            name: "HTTP Status",
+            value: abortExperiment.httpStatus,
+            input: {
+              type: "number",
+              key: "httpStatus",
+              validation: yup.number().integer().moreThan(99).lessThan(600),
+            },
+          },
+        ]}
+      />
+      <Grid container justify="center">
+        <Button text="Back" onClick={onBack} />
+        <Button text="Next" destructive onClick={onSubmit} />
+      </Grid>
+    </WizardStep>
+  );
+};
+
+const LatencyExperimentDetails = () => {
+  const { onSubmit, onBack } = useWizardContext();
+  const latencyExperimentData = useDataLayout("latencyExperimentData");
+  const latencyExperiment = latencyExperimentData.displayValue();
+  const update = (key, value) => {
+    latencyExperimentData.updateData(key, value);
+  };
+
+  return (
+    <WizardStep error={latencyExperimentData.error}>
+      <MetadataTable
+        onUpdate={update}
+        data={[
+          {
+            name: "Percent",
+            value: latencyExperiment.percent,
+            input: {
+              type: "number",
+              key: "percent",
+              validation: yup.number().integer().moreThan(-1).lessThan(101),
+            },
+          },
+          {
+            name: "Duration (ms)",
+            value: latencyExperiment.durationMs,
+            input: {
+              type: "number",
+              key: "durationMs",
+              validation: yup.number().integer().moreThan(0),
+            },
+          },
+        ]}
+      />
+      <Grid container justify="center">
+        <Button text="Back" onClick={onBack} />
+        <Button text="Next" destructive onClick={onSubmit} />
+      </Grid>
```

Comment:
```suggestion
      ```suggestion
      <ButtonGroup
        buttons={[
          {
            text: "Back",
            onClick: onBack,
            destructive: true,
          },
          {
            text: "Next",
            onClick: onSubmit,
            destructive: true,
          },
        ]}
      />
```
```